nextcloud/README.md

```diff
@@ -12,11 +12,11 @@ Nextcloud is a self-hosted file storage and synchronization service.
   </tr>
   <tr>
     <td>Version</td>
-    <td><a href="https://github.com/nextcloud/server/pull/8127">13 RC4</a></td>
+    <td><a href="https://github.com/nextcloud/server/pull/8127">13.0.1</a></td>
```

Comment:
The URL references to 13 RC4. I think it's better to substitute it for https://nextcloud.com/blog/nextcloud-13.0.1-12.0.6-and-11.0.8-available-with-improved-password-protection/.